azure-pipelines.yml

```diff
@@ -297,3 +297,27 @@ stages:
           modules:
             - hibernate-search-elasticsearch
           name: hibernate_search_elasticsearch
+
+
+  - stage: test_quickstarts_stage
+    displayName: 'Quickstart Tests'
+    dependsOn: build_artifact_for_native_stage
+    jobs:
+      - job: Test_Quickstarts
+        displayName: 'Test Quickstarts'
+        timeoutInMinutes: 45
```

Comment:
45 is definitely not enough... closer to 380 minutes.

Reply:
It’s enough in JVM mode :)

It doesn’t change the fact that we would need CI in the quickstarts themselves for native.

Reply:
If we were going to do the native builds we need to split it up like we do for our current native builds, and I don't think we have enough agents. 

I still don't like the chicken and egg problem it introduces though